src/utils/network-monitor.ts

```diff
@@ -642,20 +588,17 @@ export class NetworkMonitor {
 
   disconnectBuilder(network: string, rpcEndpoint: string, subscribe: boolean): (code: number, reason: any) => void {
     return (code: number, reason: any): void => {
-      const restart = () => {
-        this.structuredLog(
-          network,
-          `Error in websocket connection, restarting... ${webSocketErrorCodes[code]} ${JSON.stringify(reason)}`,
-        )
-        this.lastBlockJobDone[network] = Date.now()
-        this.walletNonces[network] = -1
-        this.failoverWebSocketProvider(network, rpcEndpoint, subscribe)
-      }
-
       this.structuredLog(network, `Websocket is closed. Restarting connection for ${networks[network].name}`)
-      // terminate the existing websocket
       this.ws[network].terminate()
```

Comment:
@alexanderattar i think we can remove this right? Since we no longer use the websocket rpc url

Reply:
I think we still might want the option of enabling websockets at some point